java/ql/src/Violations of Best Practice/Magic Constants/MagicConstantsString.ql

```diff
@@ -17,7 +17,7 @@ import MagicConstants
 /**
  * Holds if the string is a standard system property as defined in:
  *
- * http://docs.oracle.com/javase/7/docs/api/java/lang/System.html#getProperties()
+ * https://docs.oracle.com/javase/7/docs/api/java/lang/System.html#getProperties()
```

Comment:
Did not update this link to Java 11 because the System properties listed for that version might be different than the ones for Java 7.